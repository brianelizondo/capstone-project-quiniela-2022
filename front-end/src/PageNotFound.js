import React from "react";

function PageNotFound() {
    return (
        <h1>404 page</h1>
    );
}

export default PageNotFound;