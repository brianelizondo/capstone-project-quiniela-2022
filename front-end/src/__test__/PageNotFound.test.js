import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';

import PageNotFound from '../PageNotFound';

// Smoke Test
describe("Smoke Test", () => {
    it("renders 'PageNotFound' component without crashing", () => {
        render(<PageNotFound />);
        const textFound = screen.getByText("404 page");
        expect(textFound).toBeInTheDocument();
    });
});

// Snapshot Tests
describe("Snapshot Test", () => {
    it("'PageNotFound' matches snapshot", () => {
        const {asFragment} = render(<PageNotFound />);
        expect(asFragment()).toMatchSnapshot();
    });
});