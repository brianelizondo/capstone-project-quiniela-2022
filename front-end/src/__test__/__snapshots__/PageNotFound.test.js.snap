// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Test 'PageNotFound' matches snapshot 1`] = `
<DocumentFragment>
  <h1>
    404 page
  </h1>
</DocumentFragment>
`;
